packages/driver/src/cy/snapshots.ts

```diff
@@ -254,33 +342,51 @@ export const create = ($$: $Cy['$$'], state: StateFunc) => {
         name: string
```

Comment:
The logic in this `if` statement is probably big enough now that we should probably create a new function for it.

Reply:
addressed in 9132ea1225